config/ember-try.js

```diff
@@ -1,23 +1,24 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
 module.exports = async function () {
   return {
     scenarios: [
       {
-        name: 'ember-lts-3.16',
+        name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
-            'ember-source': '~3.16.0',
+            'ember-source': '~3.24.3',
           },
         },
       },
       {
-        name: 'ember-lts-3.20',
+        name: 'ember-lts-3.28',
```

Comment:
I think something went wrong in the latest LTS patch release. This scenario shouldn't have been part of that since the ember version in package.json is 3.28 already. I would roll back to the version in the 3.28.4 release: https://github.com/ember-cli/ember-addon-output/blob/v3.28.4/config/ember-try.js

Reply:
I always found it odd that Ember 3.28 would only test for 3.24 while they usually advertise compatibility with 2 older versions.